release/cyclone-server.yaml

```diff
@@ -60,6 +60,7 @@ _config:
             "cyclone_server_host": "0.0.0.0",
             "cyclone_server_port": 7099,
             "webhook_url": "[[ compassWebEndpoint ]]/hodor/apis/admin.devops.caicloud.io/v2",
+            "record_web_url_template": "[[ compassWebEndpoint ]]/devops/pipeline/{{.WorkflowName}}/record/{{.WorkflowRunName}}?workspace={{.ProjectName}}&tenant={{.Tenant}}",
```

Comment:
Is the [cyclone.yaml.template](https://github.com/caicloud/cyclone/blob/master/manifests/cyclone.yaml.template) need the config ?